book.md

```diff
@@ -1769,18 +1769,22 @@ most CPUs have multiple pieces of
 [branch predictor](https://en.wikipedia.org/wiki/Branch_predictor)\index{branch
 predictor} logic, such as:
 
-- A component that predicts the outcome of a conditional branch: taken or not
+- A component that predicts the outcome of a conditional
+  branch\index{Conditional branch direction predictor}: taken or not
   taken\index{taken branch}. The prediction is typically done based on the
   outcome of this and other branches in the recent past. In academic literature,
   this type of component is often called a Pattern History Table (PHT)
-  \index{Pattern History Table (PHT)}.
+  \index{Pattern History Table (PHT)}, referring to a specific implementation of
+  a conditional branch direction predictor.
 - A component that predicts the target of a taken branch, i.e. the address of
   the next instruction after a taken branch. In academic literature, such a
   component is often called a Branch Target Buffer (BTB) \index{Branch Target
-  Buffer}.
+  Buffer}, referring to a specific implementation of a branch target
```

Comment:
In contrast, as far as branch target predictors are concerned, the usage of BTB is probably widespread (universal?) enough that I don't have any strong feelings on whether it's worth mentioning that it's just a specific implementation.